mobile/client_test.go

```diff
@@ -4,149 +4,103 @@ import (
 	"testing"
 )
 
-func TestStrings_MarshalJSON(t *testing.T) {
-	t.Run("Test with non empty string array", func(t *testing.T) {
+func TestProps_Marshal(t *testing.T) {
+	t.Run("Convert a full props object", func(t *testing.T) {
```

Comment:
Btw. subtest names are supposed to be similar to test names: like Go symbols. They can be selected on the command-line in regexps: `go test -run=TestProps_Marshal/ConvertPropsFull`

Reply:
Awesome. 🦾 Thanks. It's been quite some time I am not coding in go and did not know about the practice.
This naming style is borrowed from an android or javascript world. Thanks for showing an example. 😺

Reply:
Quick googling showed that the practice is not as strict as I claimed, but the naming should still be structured instead of prose. :smile:

Reply:
> go test -run=TestProps_Marshal/ConvertPropsFull

> but the naming should still be structured instead of prose. 😄

Yes, and it also commands line friendly. I would like to follow this as well.